### ISSUE TITLE
Suggestion involving !skip

### ISSUE THREAD

**Issue by hoihoi661** [Yemasthui/basicBot#71]
Whenever wait list is full it locks it then skips... blaah blah blah.

Wouldn't it be a better idea to, put djcycle on > skip song > move dj > put djcycle off?

**Comment by xSidewinderx** [comment]
No. Lockskip is to make sure the user gets moved back to correct spot without someone else joining the waitlist to make it full then having to wait. If you used DJcycle all that does it make it so the user at the front doesn't go loop back after he djs, so it give other users a chance to join without he being automatically put back on the list. With lockskip, We are skipping the user, but giving him another chance at playing at a song without waiting for another 2hours(estimate depending on users)

**Comment by Hunchmun** [comment]
Are you talking about skip or lockskip? 

If you are actually talking about !lockskip, then yes, I would tend to agree with you that Cycle might be a more efficient way to go. I have deployed this in the bot that runs in the "Just a chill room..."

However, this probably needs to be discussed with Benzi as he knows the codebase a lot better than I do

**Comment by hoihoi661** [comment]
Well both I guess, still kind of confused by the change in !lockskippos to !skippos as now !skip and !lockskip do the same don't they?

**Comment by Dazzuh** [comment]
^ yeah
!skip is essentially now !lockskip, and !forceskip is now !skip.
